pkg/testing/ess/create_deployment_csp_configuration.yaml

```diff
@@ -13,3 +13,7 @@ aws:
   elasticsearch_conf_id: "aws.es.datahot.i3"
   elasticsearch_deployment_template_id: "aws-storage-optimized"
   kibana_instance_configuration_id: "aws.kibana.c5d"
+docker:
+  integration_server_image: "docker.elastic.co/cloud-release/elastic-agent-cloud:9.1.0-6afab230-SNAPSHOT"
+  elasticsearch_image: "docker.elastic.co/cloud-release/elasticsearch-cloud-ess:9.1.0-6afab230-SNAPSHOT"
+  kibana_image: "docker.elastic.co/cloud-release/kibana-cloud:9.1.0-6afab230-SNAPSHOT"
```

Comment:
I like this! It would be great it we could have this pinned always, then have a bot that automatically creates a PR on a new release to verify that it works before it is merged.

Reply:
That is more or less the idea, having some automation trying the latest and greatest and create a PR to bump the versions.
The version I picked (manually) is the date when main was still succeeding (Feb 24th)

Reply:
Alternatively, we can get the latest stable snapshot version in runtime instead of making automation and PR

Reply:
My preference is to use pinned versions to help with the reproducibility. The PRs loop can help to validate the changes are acceptable to be merged. That's how we can avoid surprises with any versions if we use dynamism